src/features/world/scenes/BeachScene.ts

```diff
@@ -613,29 +703,144 @@ export class BeachScene extends BaseScene {
   public handleOtherDiggersPositions() {
     // If any other players are inside of the dig area, move them to the perimeter
     this.mmoServer.state.players.forEach((player, sessionId) => {
```

Comment:
Consider making this optional chaining - `this.mmoServer?.state....`

If we run the game locally without a `VITE_ROOM_URL` this will crash the game.

<img width="1467" alt="Screenshot 2024-07-22 at 9 27 31 AM" src="https://github.com/user-attachments/assets/206043dc-bdbc-4280-afad-1552078b6c76">